src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -101,10 +76,10 @@ export const pinTokensWithBalanceAtTop = ({
 		...positiveBalances.sort(
 			(a, b) =>
 				(b.usdBalance ?? 0) - (a.usdBalance ?? 0) ||
-				b.balance - a.balance ||
+				Number(b.formattedBalance ?? 0) - Number(a.formattedBalance ?? 0) ||
```

Comment:
Can we use balance here as well?

Reply:
I tried here too, but it becomes quite verbose, since I have to use both `gt` and `lt` for the bignumbers. That part of the condition would become

```
(b.balance ?? BigNumber.from(0)).gt(a.balance ?? BigNumber.from(0)) - 
        (a.balance ?? BigNumber.from(0)).gt(b.balance ?? BigNumber.from(0))
```

but if all ok, I will add it there

Reply:
You can probably have a constant somewhere `const ZERO = BigNumber.from(0)` (or else) that should make things less verbose.